README.md

```diff
@@ -1 +1,41 @@
-# vc-module-tax
\ No newline at end of file
+# vc-module-tax
+
+## Overview
+
+The tax module is an engine for tax calculations and allows doing the following:
+
+1. Extend the custom tax providers;
+1. Tax providers registration using the code;
+1. Display the list of available tax providers on UI;
+1. Edit tax provider settings on UI;
+1. Connect the tax providers to the selected Store;
+
+## Scenarios
+
+### View Tax Provider
+
+1. Go to More->Stores->select the Store;
```

Comment:
I would remove Avalara at all and used Fixed tax provider as an example. As it is the one that is shipped with the module. And need to change the screenshosts